### ISSUE TITLE
question about training datasets

### ISSUE THREAD

**Issue by MASTERCHIEF343** [lmb-freiburg/demon#66]
Hi, I'm working on my network.And i use rgbd and mvs datasets for training.But I find that mvs isn't work very well, the test accuracy is bad.But the training loss is very well and accuracy for rgbd test datasets is similliar with you.Have you ever met that situation?
